generators/app/index.js

```diff
@@ -69,6 +55,43 @@ Have fun!`);
   }
 };
 
+function installAndRun(appPath, options) {
+  const installer = options.installer;
+
+  const parcelPath = require.resolve(".bin/parcel");
+  const indexPath = path.join(appPath, "app/index.html");
+
+  return new Promise((resolve, reject) => {
+    let installComplete = false;
+
+    say(`\nStarting development server...`);
+    const parcelProc = spawn(parcelPath, [indexPath], {
+      stdio: "inherit"
+    });
+
+    const installProc = spawn(installer, ["install"], {
+      cwd: appPath,
+      stdio: "ignore"
+    });
+
+    installProc.on("exit", () => {
+      installComplete = true;
+    });
+
+    process.once("SIGINT", () => {
+      if (!installComplete) {
+        say(`
+
+Your project is ready to go, but I wasn't able to finish installing dependencies
+in the background while you were working. You'll need to run '${installer} install'
+yourself to complete the process. The generated README.md in ${appPath}
+contains instructions for running the live server, tests, etc.
+`);
```

Comment:
We should probably also display this message (or something similar) if the install process fails.